Documentation/Assignment 2/tex/assign_2_functional.tex

```diff
@@ -0,0 +1,46 @@
+\chapter{Functional Requirements}
+
+\section{Introduction}
+In this document the requirements for assignment 2 will be stated. There will be improvements to the code, as well as new functionalities. \noindent This will be done using the \fnurl{MoSCoW Model}{https://en.wikipedia.org/wiki/MoSCoW_method}.
+
+\section{Must Have}
+\begin{itemize}
+\itemsep0em
+\item The player must be able to fall through the floor.
+\item The player must reappear after falling through the floor at the corresponding square in the ceiling.
+\item The monsters must be able to fall though the floor.
+\item The monsters must reappear after falling through the floor at the corresponding square in the ceiling.
+\item The monsters must be able to jump.
```

Comment:
This line can be deleted. The same thing with extra's is said in the line below.